src/gnoi/traceroute.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"time"
+
+	log "github.com/golang/glog"
+	system "github.com/openconfig/gnoi/system"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+func main() {
+	// Add input parameters
+	username := flag.String("username", "admin", "username for connection to gNOI")
+	password := flag.String("password", "admin", "password for connection to gNOI")
+	target := flag.String("target", "172.20.20.2:6030", "Target ip or hostname of the device running gNOI")
+	destination := flag.String("destination", "2.2.2.2", "Destination of the address to traceroute to")
```

Comment:
i think you need a `flag.Parse()` here otherwise none of these flags will get processed